docs/cog_guides/downloader.rst

```diff
@@ -0,0 +1,547 @@
+.. _downloader:
+
+==========
+Downloader
+==========
+
+This is the cog guide for the downloader cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load downloader
+
+.. _downloader-usage:
+
+-----
+Usage
+-----
+
+Install community cogs made by Cog Creators.
+
+Community cogs, also called third party cogs, are not included
+in the default Red install.
+
+Community cogs come in repositories. Repos are a group of cogs
+you can install. You always need to add the creator's repository
+using the ``[p]repo`` command before you can install one or more
+cogs from the creator.
+
+
+.. _downloader-commands:
+
+--------
+Commands
+--------
+
+.. _downloader-command-cog:
+
+^^^
+cog
+^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog 
+
+**Description**
+
+Base command for cog installation management commands.
+
+.. _downloader-command-cog-checkforupdates:
+
+"""""""""""""""""""
+cog checkforupdates
+"""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog checkforupdates 
+
+**Description**
+
+Check for available cog updates (including pinned cogs).
+
+This command doesn't update cogs, it only checks for updates.
+Use ``[p]cog update`` to update cogs.
+
+.. _downloader-command-cog-info:
+
+""""""""
+cog info
+""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog info <repo_name> <cog_name>
+
+**Description**
+
+List information about a single cog.
+
+Example:
+    - ``[p]cog info 26-Cogs defender``
+
+**Arguments**
+
+- ``<repo_name>`` The repo to get cog info from..
+- ``<cog_name>`` The cog to get info on.
+
+.. _downloader-command-cog-install:
+
+"""""""""""
+cog install
+"""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog install <repo_name> <cogs>
+
+**Description**
+
+Install a cog from the given repo.
+
+Examples:
+    - ``[p]cog install 26-Cogs defender``
+    - ``[p]cog install Laggrons-Dumb-Cogs say roleinvite``
+
+**Arguments**
+
+- ``<repo_name>`` The name of the repo to install cogs from.
+- ``<cogs>`` The cog or cogs to install.
+
+.. _downloader-command-cog-installversion:
+
+""""""""""""""""""
+cog installversion
+""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog installversion <repo_name> <revision> <cogs>
+
+**Description**
+
+Install a cog from the specified revision of given repo.
+
+Example:
+    - ``[p]cog installversion Broken-Repo e798cc268e199612b1316a3d1f193da0770c7016 cog_name``
+
+**Arguments**
+
+- ``<repo_name>`` The name of the repo to install cogs from.
+- ``<revision>`` The revision to install from.
```

Comment:
Expanding what a "revision" is could be useful since end users probably don't know much about git.

Reply:
Added an explanation of revision with a link and an image. It still makes assumptions that the user know what they're doing though.